app/src/main/java/edu/uc/kovaciad/slicetracker/ui/main/MainViewModel.kt

```diff
@@ -10,10 +10,14 @@ import com.google.firebase.firestore.FirebaseFirestoreSettings
 import edu.uc.kovaciad.slicetracker.dto.*
 
 class MainViewModel : ViewModel() {
-    var firestore: FirebaseFirestore = FirebaseFirestore.getInstance()
+    private var firestore: FirebaseFirestore = FirebaseFirestore.getInstance()
```

Comment:
Using explicit "private" visibility modifier. Set to private for now, but could change to internal if expected to use outside of class in same module